### ISSUE TITLE
Run-AlPipeline 'appFolders' because it is an empty array

### ISSUE THREAD

**Issue by dfredborg** [microsoft/navcontainerhelper#1377]
I am getting an error:
![image](https://user-images.githubusercontent.com/19800807/96408278-739f4c00-11e3-11eb-8e1e-5137e4e6d717.png)

 when I try to run the following script:
![image](https://user-images.githubusercontent.com/19800807/96408221-55395080-11e3-11eb-8831-195fd62217fc.png)

And if i try to say okay then I will just add an extra app folder so it looks like this:

 -appFolders @("app,app") `

I get the following error:

![image](https://user-images.githubusercontent.com/19800807/96408384-ac3f2580-11e3-11eb-830d-23ebc6e79995.png)

The structure of my app is as follows:

![image](https://user-images.githubusercontent.com/19800807/96408429-c973f400-11e3-11eb-977c-11572b24fc32.png)

What am I doing wrong 🤔

**Comment by freddydk** [comment]
The function accepts either an array or a string
if you had a folder (next to the app folder) called app2  it would be

-appFolders @("app","app2)
or
-appFolders "app,app2"

@("app","app2) is an array with to elements

**Comment by dfredborg** [comment]
I am still getting an error 

![image](https://user-images.githubusercontent.com/19800807/96420365-b79b4c80-11f5-11eb-95d1-b4a3b691b36e.png)

I only have one App folder so I do not really need an array

**Comment by freddydk** [comment]
Cannot repro this - tried to run exactly the same script as above (in 1.0.9)

Tries this:
```
Run-AlPipeline -pipelineName "xx" -licenseFile "C:\temp\license.flf" -baseFolder "C:\Users\freddyk\Documents\GitHub\BusinessCentralApps\BingMaps - Copy" -appFolders "app" -enablePerTenantExtensionCop -enableUICop
```
(thought it could be something with the space in the basefolder" - but it works.

**Comment by freddydk** [comment]
What is the content of your app folder above?

**Comment by dfredborg** [comment]
I have tried to run it on two different projects but with the same outcome:

![image](https://user-images.githubusercontent.com/19800807/96546773-27b7da00-12ab-11eb-8edb-fdfab3b180c8.png)

and the simple one:

![image](https://user-images.githubusercontent.com/19800807/96546814-3acaaa00-12ab-11eb-9cf9-5f5b65f1a44e.png)

The funny thing is that it worked, until it did not, so I do not know if there has been a docker update or windows update.

**Comment by freddydk** [comment]
This is not due to docker or windows updates.
Shoot me an email to freddyk at microsoft dot com and lets setup a 15 minute teams call - I would like to see this (other people might hit the same).

**Comment by dfredborg** [comment]
Unfortunately the new commit did not solve my problem ☹

![image](https://user-images.githubusercontent.com/19800807/96560214-4c697d00-12be-11eb-8831-bdd2717d897a.png)

**Comment by freddydk** [comment]
The fix is not included in any release yet (build/release process takes 2+ hours)

/Freddy

Fra: Dennis Fredborg <notifications@github.com>
Sendt: 20. oktober 2020 10:24
Til: microsoft/navcontainerhelper <navcontainerhelper@noreply.github.com>
Cc: Freddy Kristiansen <Freddy.Kristiansen@microsoft.com>; Comment <comment@noreply.github.com>
Emne: Re: [microsoft/navcontainerhelper] Run-AlPipeline 'appFolders' because it is an empty array (#1377)


Unfortunately the new commit did not solve my problem ☹

[image]<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F19800807%2F96560214-4c697d00-12be-11eb-8831-bdd2717d897a.png&data=04%7C01%7Cfreddy.kristiansen%40microsoft.com%7C9141b001828b43020c6508d874d18112%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637387790453018524%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=to4HL2gtalAUSDi4cVieDUqJmFigkyb1wMoWVOoq9f0%3D&reserved=0>

**Comment by freddydk** [comment]
Shipped in 1.0.10-preview224

**Comment by dfredborg** [comment]
I works thanks for the help 😊

**Comment by freddydk** [comment]
Fix shipped in 1.0.10
